### PR TITLE
Better examples and more complete documentation around the task.setvariable logging command

### PR DESCRIPTION
**Pull request by john-landgrave**

### CODE REVIEW

**Comment by PRMerger20** [comment]
@john-landgrave : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by juliakm** [comment]
john-landgrave Thank you for these very helpful additions!

#sign-off

File: docs/pipelines/process/set-variables-scripts.md [hunk 1]
```diff
@@ -191,13 +191,13 @@ jobs:
 ```
 
 ---
-## Set an output variable in a future job
+## Set an output variable for use in future jobs
 
-When you use output variables across jobs, you'll reference them with `dependencies`. Learn more about [dependencies](expressions.md). 
+When you use output variables across jobs, you'll reference them with `dependencies`. The syntax for access an output variable in a future job or stage varies based on the relationship between the setter and consumer of the variable. Learn about each case in [dependencies](expressions.md#dependencies).
```
Comment (ShannonLeavitt) [inline docs/pipelines/process/set-variables-scripts.md:196]:
```suggestion
When you use output variables across jobs, you'll reference them with `dependencies`. The syntax for accessing an output variable in a future job or stage varies based on the relationship between the setter and consumer of the variable. Learn about each case in [dependencies](expressions.md#dependencies).
```

**Comment by PRMerger3** [comment]
@ShannonLeavitt : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Pull request merged**